sklearn/cluster/tests/test_mean_shift.py

```diff
@@ -55,6 +55,36 @@ def test_mean_shift():
     assert_equal(n_clusters_, n_clusters)
 
 
+def test_mean_shift_negative_bandwidth():
+    bandwidth = -1
+    ms = MeanShift(bandwidth=bandwidth)
+    msg = \
```

Comment:
Use parentheses to enclose expressions and split them over multiple lines rather than using \ for line continuation

Reply:
@jnothman this comment is not clear will following statement work?

 msg = "bandwidth needs to be greater than zero or None," \
        "            got -1.000000"

Reply:
This will:
```
msg = ("bandwidth needs to be greater than zero or None," 
       " got -1.000000")
```

Reply:
changed